irods_consortium_continuous_integration_test_hook.py

```diff
@@ -35,6 +35,7 @@ def main():
         irods_python_ci_utilities.subprocess_get_output(['sudo', 'su', '-', 'irods', '-c',
             f'python3 scripts/run_tests.py --xml_output --run_s {test} 2>&1 | tee {test_output_file}; exit $PIPESTATUS'],
             check_rc=True)
+        irods_python_ci_utilities.subprocess_get_output(['python3', '-m',  'pip',  'install', 'python-irodsclient'], check_rc=True)
```

Comment:
Let's put this in the if-block that starts on line 21 since it's a "setup" step.

Maybe we should pin the version number, too. Let's go with 1.1.9.